SWLOR.Game.Server/ChatCommand/SetScale.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using SWLOR.Game.Server.NWN;
+using SWLOR.Game.Server.ChatCommand.Contracts;
+using SWLOR.Game.Server.Enumeration;
+using SWLOR.Game.Server.GameObject;
+using SWLOR.Game.Server.NWN.Enum;
+
+namespace SWLOR.Game.Server.ChatCommand
+{
+    [CommandDetails("Sets Scale.", CommandPermissionType.DM | CommandPermissionType.Admin)]
+    public class SetScale : IChatCommand
+    {
+        public void DoAction(NWPlayer user, NWObject target, NWLocation targetLocation, params string[] args)
+        {
+
+            float value = float.Parse(args[0]);
```

Comment:
You need more error checking on this. If someone does /setScale boogaloo, you're going to get an exception because boogaloo can't be converted to a float.